app/graphql/mutations/users/update_user.rb

```diff
@@ -0,0 +1,29 @@
+module Mutations
+  module Users
+    class UpdateUser < Mutations::BaseMutation
+      argument :params, Types::Input::Users::UpdateUserType, required: true
+
+      field :user, Types::UserType, null: false
+
+      def resolve(params:)
+        user_params = Hash params
+        if valid_user_id?(user_params[:id])
+          user = User.find(user_params[:id])
+          user.update!(user_params)
+
+          { user: user }
+        elsif !valid_user_id?(user_params[:id])
```

Comment:
Could leave it as else instead of elsif, because it will jump in there if the first predicate is false.